package.json

```diff
@@ -52,13 +50,17 @@
     "html-webpack-plugin": "^1.7.0",
     "json-loader": "^0.5.4",
     "postcss-loader": "^0.8.0",
+    "react-hot-loader": "^1.3.0",
+    "react-redux": "^4.0.0",
     "react-transform-catch-errors": "^1.0.0",
     "react-transform-hmr": "^1.0.1",
     "redbox-react": "^1.2.0",
+    "redux": "^3.0.2",
     "style-loader": "^0.13.0",
     "url-loader": "^0.5.7",
     "webpack": "^1.12.9",
     "webpack-dev-middleware": "^1.4.0",
+    "webpack-dev-server": "^1.14.0",
```

Comment:
this boilerplate uses `webpack-dev-middleware`. so `webpack-dev-server` is unnecessary here